<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqPluginDialog</class>
 <widget class="QDialog" name="pqPluginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugin Manager</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="HelpText">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="remoteGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Remote Plugins</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="2" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="2" colspan="5">
       <widget class="pqPluginTreeWidget" name="remotePlugins">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="loadRemote">
        <property name="text">
         <string>Load New ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="removeRemote">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="loadSelected_Remote">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="localGroup">
     <property name="title">
      <string>Local Plugins</string>
     </property>
     <layout class="QGridLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="3" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="2" colspan="4">
       <widget class="pqPluginTreeWidget" name="localPlugins">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="removeLocal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="loadLocal">
        <property name="text">
         <string>Load New ...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="loadSelected_Local">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqPluginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqPluginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <customwidgets>
  <customwidget>
    <class>pqPluginTreeWidget</class>
    <extends>QTreeWidget</extends>
    <header>pqPluginTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>remotePlugins</tabstop>
  <tabstop>loadRemote</tabstop>
  <tabstop>loadSelected_Remote</tabstop>
  <tabstop>removeRemote</tabstop>
  <tabstop>localPlugins</tabstop>
  <tabstop>loadLocal</tabstop>
  <tabstop>loadSelected_Local</tabstop>
  <tabstop>removeLocal</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
</ui>
